### DISCUSSION TITLE
My deploy of Nebari local (in kind) says "service unavailable"

### DISCUSSION THREAD

**Discussion by verhulstm** [nebari-dev/nebari#2884] [Community]
I am looking for trouble shooting advice on this. maybe i missed a stop? thanks :)

**Comment by marcelovilla** [comment]
Hey @verhulstm! What OS are you using to deploy Nebari? And could you paste the output you got here?

**Comment by viniciusdc** [comment]
Hi @verhulstm, I'm following up to see if you’re still experiencing the abovementioned issue. If so, please share a screenshot of the error and your `nebari-config.yaml` file. This will help us replicate and investigate the problem.

**Comment by marcelovilla** [comment]
@verhulstm I'm closing this discussion due to inactivity. Feel free to reopen it if you're still experiencing this issue.
